hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerBalancerStartSubcommand.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hdds.scm.cli;
+
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.scm.client.ScmClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+import java.io.IOException;
+
+/**
+ * Handler to start container balancer.
+ */
+@Command(
+    name = "start",
+    description = "Start ContainerBalancer",
+    mixinStandardHelpOptions = true,
+    versionProvider = HddsVersionProvider.class)
+public class ContainerBalancerStartSubcommand extends ScmSubcommand {
+
+  private static final Logger LOG =
+      LoggerFactory.getLogger(ContainerBalancerStartSubcommand.class);
+
+  @CommandLine.Option(names = {"-t", "--threshold"},
+      description = "Threshold target whether the cluster is balanced",
+      defaultValue = "0.1",
```

Comment:
There would also be a corresponding configuration for threshold and other parameters. How do we make sure that the configuration defaults are in sync with the default values in command line?
I think it might be better to not include a default value here or have an invalid value by default.

Reply:
Let's say user has set a configuration for threshold to be 0.05. If user does not specify threshold as parameter during start, threshold of 0.05 should be used in balancer rather than 0.1.

Reply:
thanks @lokeshj1703 for the review ! will remove the default value.

Reply:
updated, PTAL!
there is no any logic code here , just send a command , so maybe no need to add unit test , just like replication manager cli. what is more ,  acceptance test can be added after we complete container balancer,  because by then the acceptance will make more sense! what do you think about？

Reply:
It would be good to add a base test to test the CLI. For example simple test to start balancer and check status. Additional parameters when added can be tested in the UT as well.
Some UTs where it is done - TestContainerOperations, TestDecommissionSubCommand etc.

Reply:
ok, will add test